subscriptions.go

```diff
@@ -101,6 +133,24 @@ func (s *SubscriptionAPI) List() ([]*Subscription, error) {
 	return subscriptions.Items, nil
 }
 
+// ListFilter returns all subscriptions that consistent with the `SubscriptionParameter`s
+// DEPRECATED: will be replaced in version 2. Use `List()` instead
```

Comment:
Since the `List` method does not work as expected, just write:
```go
// DEPRECATED: will be replaced in version 2.
```
Without referencing the `List` method.